<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Аквариумный свет</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Таймеры</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="cbEnTimers">
            <property name="text">
             <string>Работа по таймерам</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="tvAllTimers">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="dragDropOverwriteMode">
             <bool>false</bool>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="verticalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="horizontalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pbAddTimer">
              <property name="text">
               <string>Добавить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbExport">
              <property name="text">
               <string>Экспорт</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbImport">
              <property name="text">
               <string>Импорт</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbDelTimer">
              <property name="text">
               <string>Удалить</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Ручная настройка ленты</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>150</height>
             </size>
            </property>
            <property name="title">
             <string>Светодиоды в ленте</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label">
                 <property name="minimumSize">
                  <size>
                   <width>62</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>62</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Кол-во</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="sbCount">
                 <property name="maximum">
                  <number>300</number>
                 </property>
                 <property name="value">
                  <number>300</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="minimumSize">
                  <size>
                   <width>62</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>62</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Начинать с</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="sbStart">
                 <property name="maximum">
                  <number>300</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="minimumSize">
                  <size>
                   <width>62</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>62</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Размер ряда</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="sbElementsInRow">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>300</number>
                 </property>
                 <property name="value">
                  <number>1</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbMakeMatrix">
            <property name="text">
             <string>Построить матрицу</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbDelMatrix">
            <property name="text">
             <string>Очистить матрицу</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QProgressBar" name="pbarUploading">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Матрица ленты</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QGridLayout" name="ledGrid"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actRefresh"/>
    <addaction name="separator"/>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actAbout"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Время</string>
    </property>
    <addaction name="actGetTime"/>
    <addaction name="actSyncTime"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Режим</string>
    </property>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>Светодиоды</string>
     </property>
     <addaction name="actLedOff"/>
     <addaction name="actLedOn"/>
    </widget>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>Реле</string>
     </property>
     <addaction name="actRelay0"/>
     <addaction name="actRelay1"/>
    </widget>
    <widget class="QMenu" name="menu_7">
     <property name="title">
      <string>Вручную</string>
     </property>
     <addaction name="actSetUserColor"/>
    </widget>
    <addaction name="actSunrise"/>
    <addaction name="actSunset"/>
    <addaction name="menu_7"/>
    <addaction name="actCustomMode"/>
    <addaction name="menu_5"/>
    <addaction name="menu_6"/>
   </widget>
   <widget class="QMenu" name="menu_8">
    <property name="title">
     <string>Подключение</string>
    </property>
    <addaction name="actIPPort"/>
    <addaction name="actConnect"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_8"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actImport">
   <property name="text">
    <string>Импорт</string>
   </property>
  </action>
  <action name="actExport">
   <property name="text">
    <string>Экспорт</string>
   </property>
  </action>
  <action name="actExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actNew">
   <property name="text">
    <string>Новый</string>
   </property>
  </action>
  <action name="actAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actGetTime">
   <property name="text">
    <string>Получить</string>
   </property>
  </action>
  <action name="actSyncTime">
   <property name="text">
    <string>Синхронизировать</string>
   </property>
  </action>
  <action name="actSunrise">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Рассвет</string>
   </property>
  </action>
  <action name="actSunset">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Закат</string>
   </property>
  </action>
  <action name="actLedOff">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Выключить</string>
   </property>
  </action>
  <action name="actLedOn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Включить</string>
   </property>
  </action>
  <action name="actRelay0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </action>
  <action name="actRelay1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="actCustomMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Настраиваемый</string>
   </property>
  </action>
  <action name="actSetUserColor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Задать цвет</string>
   </property>
  </action>
  <action name="actRefresh">
   <property name="text">
    <string>Обновить</string>
   </property>
  </action>
  <action name="actIPPort">
   <property name="text">
    <string>Указать IP:Port</string>
   </property>
  </action>
  <action name="actConnect">
   <property name="text">
    <string>Подключиться</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
